### ISSUE TITLE
[A11y] Search Results - Grid - Parsing

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1960]
### Operating System

Windows

### Browser

Edge

### Affected page

Search

### Admin user?

Yes

### Issue Description

There's a closing "p" tag that the w3c validator claims is incorrect.  Tidied but unmodified HTML:

```html
<p class="range subsection slider_js">
  <div style="display:none">
    Current results range from <span class="min">1500</span> to <span class="max">2021</span>
  </div>
  <div class="slider slider-horizontal" style="width: 100%;">
    <div class="slider-track">
      <div class="slider-selection" style="left: 0%; width: 100%;"></div>
      <div class="slider-handle round" style="left: 0%;"></div>
      <div class="slider-handle round" style="left: 100%;"></div>
    </div>
    <div class="tooltip top hide" style="top: -24px; left: 0px;">
      <div class="tooltip-arrow"></div>
      <div class="tooltip-inner">1500 : 2022</div>
    </div>

    <input type="text" data-slider-placeholder="true" style="width: 100%;">
  </div>
</p>
```

The HTML looks like it's fine, but it turns out a div cannot be inside a paragraph, and "p" tags are self-closing, so the browser will actually treat it like you wrote this:

```html
<p class="range subsection slider_js"></p>
  <div style="display:none">
  ...
  </div>
</p>
```

Note the injected closing "p" tag.

### Suggested Fix

Switch the surrounding tag to a div if possible?  Otherwise, just close the paragraph on the line that opens it so the HTML is the same as the DOM ends up.

This is probably not a major problem since browsers guess at the DOM anyway, but it's an easy thing for a drive-by accessibility auditor to catch.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Verify the HTML is valid

**Comment by jechols** [comment]
So... the specific issue highlighted in this ticket was indeed fixed, but now there's a new one: this HTML is seen twice in the page: `<span class="sr-only" id="search-per-page">` It appears that the two are used for different responsive views, so I'm not sure how best to fix such a thing....
